extensions/amp-form/0.1/amp-form.js

```diff
@@ -274,8 +274,13 @@ export class AmpForm {
   renderTemplate_(data = {}) {
     const container = this.form_.querySelector(`[${this.state_}]`);
     if (container) {
+      const randId = `rendered-message-${Math.random() * 1000}`;
```

Comment:
Updated the code with the simple counter idea, let me know if we should just go the ID generator route instead.